include/unifex/connect_awaitable.hpp

```diff
@@ -81,7 +81,7 @@ class _sender_task<Receiver>::type {
         bool await_ready() noexcept {
           return false;
         }
-        void await_suspend(coro::coroutine_handle<promise_type>) {
+        void await_suspend(coro::coroutine_handle<promise_type>) noexcept(std::is_nothrow_invocable_v<Func>) {
```

Comment:
MSVC is complaining that `Func` is "not a symbol", or something, which is bananas because it's clearly a symbol that's in scope.

There must be some bug with defining a function-local struct with traits that depend on the surrounding function's template parameters.  Some possible workarounds:
 - just before `struct awaiter {`, declare a `static constexpr bool isNothrowInvocable = std::is_nothrow_invocable_v<Func>;` and change this line to be `noexcept(isNothrowInvocable)`
 - move the definition of `awaiter` out of `yield_value`, which requires making it a class template like this:

```
template <typename Func>
struct awaiter final {
  // we want a forwarding reference here
  Func&& func_;

  // the rest as before
};

template <typename Func>
auto yield_value(Func&& func) noexcept {
  return awaiter<Func&&>{static_cast<Func&&>(func)};
}
```